packages/ses/src/tame-faux-data-properties.js

```diff
@@ -0,0 +1,124 @@
+import {
+  getOwnPropertyDescriptor,
+  apply,
+  defineProperty,
+  toStringTagSymbol,
+} from './commons.js';
+
+/**
+ * Exported for convenience of unit testing. Harmless, but not expected
+ * to be useful by itself.
+ *
+ * @param {any} obj
+ * @param {string|symbol} prop
+ * @param {any} expectedValue
+ * @returns {boolean}
```

Comment:
Consider documenting the meaning of the return value. “The return value is a diagnostic for tests and ignored in the course of `repairIntrinsics()` or `lockdown()`. Returns whether all tests indicated that the property was a getter/setter pair intended to make the property writable objects that inherit the property on their prototype chain, and that it was (possibly temporarily) converted back to a data property.”

Possibly also: “Throws if repairs or lockdown should throw.”

Reply:
Your first paragraph, I think I've done with other words.

> Possibly also: “Throws if repairs or lockdown should throw.”

I don't understand this. Optimistically I'm going to resolve anyway. Please let me know if I'm missing something here, even if I've merged by the time you see this.